src/lib/pages/start-workflow.svelte

```diff
@@ -0,0 +1,269 @@
+
+  let workflowId = '';
+  let taskQueue = '';
+  let workflowType = '';
+  let input = '';
+  let inputRetrieved = 0;
+
+  let initialWorkflowId = '';
+  let initialWorkflowType = '';
+  let initialInput = '';
+
+  let error = '';
+  let pollerCount: undefined | number = undefined;
+  let viewAdvancedOptions = false;
+
+  let searchAttributes: SearchAttributeInput[] = [];
+
+  $: taskQueueParam = $page.url.searchParams.get('taskQueue');
+
+  onMount(() => {
+    workflowId = $page.url.searchParams.get('workflowId') || '';
+    taskQueue = $page.url.searchParams.get('taskQueue') || '';
+    workflowType = $page.url.searchParams.get('workflowType') || '';
+    initialWorkflowId = $page.url.searchParams.get('workflowId') || '';
+    initialWorkflowType = $page.url.searchParams.get('workflowType') || '';
+
+    if (initialWorkflowId || initialWorkflowType) {
+      getInitialValues(initialWorkflowId, initialWorkflowType);
+    }
+  });
+
+  const handleInputChange = (event: CustomEvent<string>) => {
+    input = event.detail;
+  };
+
+  const onWorkflowStart = async () => {
+    try {
+      error = '';
+      await startWorkflow({
+        namespace,
+        workflowId,
+        taskQueue,
+        workflowType,
+        input,
+        searchAttributes,
+      });
+      goto(routeForWorkflows({ namespace }));
+    } catch (e) {
+      error = e?.message || 'Error start Workflow';
+    }
+  };
+
+  const generateRandomWorkflowId = () => {
+    workflowId = uuid4();
+    updateQueryParameters({
+      parameter: 'workflowId',
+      value: workflowId,
+      url: $page.url,
+      allowEmpty: true,
+    });
+  };
+
+  const checkTaskQueue = async (queue: string) => {
+    if (queue) {
+      const { pollers } = await getPollers({ namespace, queue });
+      pollerCount = pollers.length;
+    }
+  };
+
+  const getInitialValues = async (id: string, type: string) => {
+    const initialValues = await fetchInitialValuesForStartWorkflow({
+      namespace,
+      workflowId: id,
+      workflowType: type,
+    });
+    initialInput = initialValues.input;
+    input = initialInput;
+    inputRetrieved = Date.now();
+    if (initialValues?.searchAttributes) {
+      const customSAKeys = Object.keys($customSearchAttributes);
+      Object.entries(initialValues.searchAttributes).forEach(([key, value]) => {
+        if (customSAKeys.includes(key)) {
+          searchAttributes = [
+            ...searchAttributes,
+            { attribute: key, value: String(value) },
+          ];
+        }
+      });
+      viewAdvancedOptions = true;
+    }
+  };
+
+  const onInputChange = (e: Event, parameter: string) => {
+    const value = (e.target as HTMLInputElement).value;
+    updateQueryParameters({
+      parameter,
+      value,
+      url: $page.url,
+      allowEmpty: true,
+    });
+  };
+
+  const onUpload = (uploadInput: string) => {
+    input = uploadInput;
+    inputRetrieved = Date.now();
+  };
+
+  $: enableStart =
```

Comment:
Do we want to check `pollerCount` here as well? Or is it fine to start even if Task Queue is inactive?

![Screenshot 2024-06-24 at 11 13 42 AM](https://github.com/temporalio/ui/assets/15069288/a8db5b53-9183-40bb-9e2e-e30e32a91bc7)

Reply:
I think we want to still allow it (maybe worker is starting later and they don't care), but good call out. We can see what users think